### PR TITLE
Firmware updates

### PR DESCRIPTION
**Pull request by wsh32**

### CODE REVIEW

**Review by wsh32 — COMMENTED** [review]
good stuff isabel! mostly minor changes and cleanup. some of the stuff i noted in the state machine file is kinda nitpicky but i realized after the fact that it was only written for one axis.

File: src/airhockey_firmware/src/airhockey_firmware_one_axis/airhockey_firmware_one_axis.ino [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    x_stepper.setAccelerationInMillimetersPerSecondPerSecond(10.0);
+    x_stepper.setCurrentPositionInMillimeters(0);
+
+    nh.initNode();
+    nh.advertise(position_feedback_publisher);
+    nh.subscribe(position_command_subscriber);
+}
+
+void loop() {
+  position_feedback_msg.data = x_stepper.getCurrentPositionInMillimeters();
```
[the remark this answers is not in the document]
Comment (iserrato) [reply src/airhockey_firmware/src/airhockey_firmware_one_axis/airhockey_firmware_one_axis.ino:46]: included

File: src/airhockey_firmware/src/airhockey_firmware_one_axis/airhockey_firmware_one_axis.ino [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ros::Subscriber<std_msgs::Int16> striker_position_subscriber(
+    "/arduino/command/striker_fb", &striker_position_callback);
+
+void position_command_callback(const std_msgs::Int16& position_cmd) {
+    delta_pos = position_cmd.data - x_stepper.getCurrentPositionInMillimeters();
+    x_stepper.moveToPositionInMillimeters(delta_pos);
+}
+
+void striker_position_callback(const std_msgs::Int16& striker_cmd) {
+  x_stepper.setCurrentPositionInMillimeters(striker_cmd.data);
```
[the remark this answers is not in the document]
Comment (iserrato) [reply src/airhockey_firmware/src/airhockey_firmware_one_axis/airhockey_firmware_one_axis.ino:29]: done

File: src/airhockey_firmware/src/airhockey_state_machine/airhockey_firmware.h [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+#include <Arduino.h>
+#include <std_msgs/Int16.h>
+#include <geometry_msgs/PointStamped.h>
+
+geometry_msgs::PointStamped position_command_msg;
+std_msgs::Int16 position_feedback_msg;
+
+void position_command_callback(const geometry_msgs::PointStamped& position_cmd);
+
```
[the remark this answers is not in the document]
Comment (iserrato) [reply src/airhockey_firmware/src/airhockey_state_machine/airhockey_firmware.h:12]: done

**Comment by iserrato** [comment]
notes:
- publishing data over ROS takes time and causes motion to be very slow
- can delete all the comments if we decide we don't need them for now
- removed print statements from flexystepper library

**Pull request merged**